omnibus/files/server-ctl-cookbooks/infra-server/attributes/default.rb

```diff
@@ -317,6 +317,9 @@
 default['private_chef']['opscode-erchef']['search_queue_mode'] = 'batch'
 default['private_chef']['opscode-erchef']['search_batch_max_size'] = '5000000'
 default['private_chef']['opscode-erchef']['search_batch_max_wait'] = '10'
+default['private_chef']['opscode-erchef']['search_auth_enabled'] = false
+default['private_chef']['opscode-erchef']['search_auth_username'] = "admin"
+default['private_chef']['opscode-erchef']['search_auth_password'] = "admin"
```

Comment:
The assumption here is that the user is manually created in the existing es setup. So this should not be a security concern.